torchvision/models/detection/retinanet.py

```diff
@@ -557,8 +546,24 @@ def forward(self, images, targets=None):
             # compute the losses
             losses = self.compute_loss(targets, head_outputs, anchors)
         else:
+            # recover level sizes
+            feature_sizes_per_level = [x.size(2) * x.size(3) for x in features]
+            HW = 0
+            for v in feature_sizes_per_level:
+                HW += v
+            HWA = head_outputs['cls_logits'].size(1)
+            A = HWA // HW
+            feature_sizes_per_level = [hw * A for hw in feature_sizes_per_level]
```

Comment:
nit: maybe `num_anchors_per_level` is more accurate as a name? It's also the same name we use in RPN https://github.com/pytorch/vision/blob/a9c78f1b017d912fa9583ae6d93bff0a3807c29b/torchvision/models/detection/rpn.py#L336 so might be good for consistency

Reply:
Good call. I did not know that a similar snippet existed elsewhere. I'll make the change. Also note that for the rest of the variable names come from notation/comments included in earlier sections:

https://github.com/pytorch/vision/blob/b639ec01ac9fcd46fb33a360c289f234b591b12c/torchvision/models/detection/retinanet.py#L142-L146

https://github.com/pytorch/vision/blob/b639ec01ac9fcd46fb33a360c289f234b591b12c/torchvision/models/detection/retinanet.py#L233-L237